mitosheet/src/mito/components/taskpanes/Merge/MergeKeysSelection.tsx

```diff
@@ -114,6 +124,12 @@ const MergeKeysSelectionSection = (props: {
                     {props.error}
                 </p>    
             }
+            {
+                nonExistentMergeKeys.length > 0 &&
+                <p className='text-color-error'>
+                    {`The column ${nonExistentMergeKeys.join(', ')} does not exist in this sheet anymore. Delete it to make this merge valid. `}
```

Comment:
Should say which sheet it's missing from, especially because often times the keys have the same name.

Reply:
(This should easy, given that you have the variables separate already).